tests/fixtures/keycloak_connection.py

```diff
@@ -0,0 +1,82 @@
+"""Module which contains constant test data related to certain keycloak connection requests."""
+
+from bson.objectid import ObjectId
+
+TEST_KEYCLOAK_USER_ID = ObjectId()
```

Comment:
@Edison-Stuart I'd double check, but I believe Keycloak is going to hand out UUIDs (`uuid4`), not ObjectIds. Won't break the tests, but it may be useful to work with the exact datatypes that will be present during normal operation.

Reply:
Okay yeah this makes sense. Some of the same reasons we want our fixture data to resemble the real thing as closely as possible.